code/index.html

```diff
@@ -4,18 +4,225 @@
   <head>
     <meta charset="utf-8" />
```

Comment:
HI! I'm Estefanía and I'll be taking a look at your code so that we can leave it as functional and shiny as possible.👋😀